great-idea/css/index.css

```diff
@@ -49,18 +49,165 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+	display: flex;
+	flex-direction: column;
+	width: 880px;
+	margin:  0 auto;
+
 }
 
 h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.nav-container {
+	height: 40px;
+	width: 880px;
+	position: fixed;
+	/*z-index: 0; */
+	background: darkblue;
+	top: 0;
+	display: flex;
+	justify-content: space-between;
+}
+
+.navbar_top {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+a{
+	padding-right: 50px;
+}
+
+.navbar_top a{
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	color: #BABABA;
+	text-decoration: none;
+	font-family: 'Raleway', sans-serif;
+	font-weight: bolder;
+}
+
+.navbar_top a:hover {
+	font-size: 16px;
+	color: white;
+}
+
+.logoIMG {
+	width: 191.78px;
+	height: 40px;
+	background: lightgrey;
+	text-align: center;
+}
+
+.header {
+	display: flex;
+	justify-content: space-evenly;
+	border-bottom: 2px solid black;
+  padding-bottom: 430px;
+	padding-top: 50px;
+	margin: 40px 0 30px 0;
+}
+
+.banner {
+	display: flex;
+	width: 400px;
+	height: 300px;
+	flex-direction: column;
+	text-align: center;
+	margin: 60px 0 0 40px;
+}
```

Comment:
some weird alignment going on in this banner. I would avoid styling in your html when you can, that caused a bit of confusion. The width you set to the h1 is choking your content a bit causing text over flow which is throwing off your text align. It seems that you had that width set to get the text on separate lines. Because of the issues caused by text overflow, I would strongly advise against this. One solution would be to use the <br> tag between words. This will cause a carriage return. <br> tags should be used sparingly and you should use your css as much as possible, but this is one of those rare occasions where it is appropriate.